spec/controllers/admin/facility_groups_controller_spec.rb

```diff
@@ -164,49 +144,6 @@
 
       expect(response).to have_http_status(:bad_request)
     end
-  end
-
-  describe "PUT #update with regions_prep enabled" do
```

Comment:
This was a duplicated block with and without `regions_prep`, now everything is for enabled in the original block.

Reply:
👍 ⚡